expression/builtin_math.go

```diff
@@ -810,16 +810,14 @@ func (b *builtinAtanSig) eval(row []types.Datum) (d types.Datum, err error) {
 		x, err := args[1].ToFloat64(sc)
 		if err != nil {
 			d.SetFloat64(math.Atan(y))
-			// FIXME: Trigger a warning perhaps.
```

Comment:
But this comments `Trigger a warning perhaps` not make sense here. For your case here you just use the err as a judging condition, but you never use it's value as it's a local variable from line 810-815